Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -34,12 +31,35 @@ def setUp(self):
             self.workspace_name = config["workspace_name"]
             self.api_key = config["api_key"]
             self.alert_id = config["alert_id"]
+            self.tenant_id = config["tenant_id"]
+            self.client_id = config["client_id"]
+            self.client_secret = config["client_secret"]
+            self.resource_url = config["resource_url"]
+            self.scope = config["scope"]
```

Comment:
Can you be a little verbose in the naming of 'scope' as a config variable? From the name I'm able to understand what it's supposed to mean.

Reply:
Is this missed?

Reply:
This is still not resolved.

Reply:
there is no way to add comment in json file.
It's commented here:
    Solutions/CohesitySecurity/Tests/az.py:242         - scope (str): The desired scope of permissions for the access token.

and it's part of request body, better to keep current name.    
    Solutions/CohesitySecurity/Tests/az.py:251     response = requests.post(
    Solutions/CohesitySecurity/Tests/az.py:258             "scope": scope,